core/domain/user_domain.py

```diff
@@ -345,3 +348,22 @@ def validate(self):
             raise utils.ValidationError(
                 'Expected can_review_questions to be a boolean value, '
                 'found: %s' % type(self.can_review_questions))
+
+
+class ModifiableUserDataV1(python_utils.OBJECT):
+    """Domain object to represent change in allowed attributes for UserSettings
+    and UserAuthDetails domain object.
+    """
+
+    def __init__(
+            self, display_alias, pin, preferred_language_codes,
+            preferred_site_language_code, preferred_audio_language_code,
+            user_id=None):
```

Comment:
Add Args section. Be sure to mention that user_id is str|None and explain when exactly it is None.

Reply:
Done now.